hardhat.config.ts

```diff
@@ -134,12 +134,12 @@ const config: HardhatUserConfig = {
     polygon: "",
     // tableland testnet testnets
     "ethereum-rinkeby": "0x30867AD98A520287CCc28Cde70fCF63E3Cdb9c3C", // deprecating: do not upgrade!
-    "ethereum-goerli": "",
-    "optimism-kovan": "",
-    "polygon-mumbai": "",
+    "ethereum-goerli": "0xa4b0729f02C6dB01ADe92d247b7425953d1DbA25",
+    "optimism-kovan": "0xf9C3530C03D335a00163382366a72cc1Ebbd39fF",
+    "polygon-mumbai": "0x70364D26743851d4FE43eCb065811402D06bf4AD",
     // tableland staging testnets
     "ethereum-rinkeby-staging": "0x847645b7dAA32eFda757d3c10f1c82BFbB7b41D0", // deprecating: do not upgrade!
-    "optimism-kovan-staging": "",
+    "optimism-kovan-staging": "0x322F01e81c38B4211529f334864fA630F6aeA408",
```

Comment:
How come there is no openzeppelin files for this one? I recall this from the past, does it overwrite the other one?

Reply:
Ah shoot, it does. We probably don't want that do we? One way to do that is to deploy the staging one first, rename the file, and then redeploy the non-staging one?

Reply:
they are both in that file... two proxies, one implementation (so far)

Reply:
Oh rad. Good idea.